pages/guideforms.py

```diff
@@ -500,6 +500,16 @@ def validate(self, value):
         ('If your feature was available as an origin trial, link to a summary '
          'of usage and developer feedback. If not, leave this empty.')),
 
+    'anticipated_spec_changes': forms.CharField(
+        required=False, label='Anticipated spec changes',
+        widget=forms.Textarea(
+            attrs={'rows': 4, 'cols': 50, 'maxlength': 500,
+                   'placeholder': 'https://\nhttps://'}),
+        help_text=
+        ('Please list known spec issues whose resolution may introduce '
```

Comment:
@glenrob gave some feedback on Monday that the text I proposed was unclear to non-insiders, e.g. my wording of "spec issues" was vague, and he wasn't sure what the point of the question was. How about:

"Open questions about a feature may be a source of future web compat or interop issues. Please list open issues (e.g. links to known github issues in the project for the feature specification) whose resolution may introduce web compat/interop risk (e.g., changing to naming or structure of the API in a non-backward-compatible way)."

The purpose of the first sentence is to make clear the *why* of this question, and the secnod is the what, but with a bit more examples.